models/articles.models.js

```diff
@@ -36,3 +36,23 @@ exports.fetchAllArticleComments = (article_id) => {
       return res.rows;
     });
 };
+
+exports.addCommentData = (addComment, article_id) => {
+  if (
+    addComment.author === undefined ||
+    addComment.body === undefined ||
+    article_id === undefined
+  ) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  } else {
+    return db
+      .query(
+        `INSERT INTO comments (author, body, article_id, created_at, votes)
+        VALUES ($1, $2, $3, $4, $5) RETURNING*;`,
+        [addComment.author, addComment.body, article_id, new Date(), 0]
```

Comment:
The votes and created_at property values are created automatically, if you look in the seed file you will see that these have default values attached, so no need to input them here 👍